Apps/Contoso.iOS.Puppet/ModulePages/CrashesController.cs

```diff
@@ -115,5 +116,10 @@ static Exception ValidateLength()
         {
             await FakeService.DoStuffInBackground();
         }
+
+        partial void NativeCrash()
+        {
+            NSNull.Null.PerformSelector(new ObjCRuntime.Selector("isEqualToString:"));
```

Comment:
Wrap it to try catch since it is test for handled exceptions

Reply:
```suggestion
            try
            {
                        NSNull.Null.PerformSelector(new ObjCRuntime.Selector("isEqualToString:"));
            } catch (Exception ex) {
                        Crashes.TrackError(ex);
            }
```